README.md

```diff
@@ -1216,7 +1224,10 @@ VSC8552; this is indicated with `--` in the relevant table positions.
 
 ### `humility openocd`
 
-This command launches OpenOCD based on the config file in a build archive
+This command launches OpenOCD based on the config file in a build archive,
+which then allows one to connect with either GDB or directly via telnet.
+If the intention is to only run GDB, note that `humility gdb --run-openocd`
+will both run OpenOCD and run a foreground GDB that is connected to it.
```

Comment:
unrelated change

Reply:
Yea, it seems that the base readme hadn't been regenerated in a bit.  Upstream has had the same problem on some of their recent PRs.
I don't know if it is worth spinning into its own PR, but always can.

Reply:
No, I think this is fine.